docs/modules/plugin/pages/goals/auto-refresh.adoc

```diff
@@ -0,0 +1,93 @@
+[[auto-refresh-goal]]
```

Comment:
Please use the shorthand anchor `[#auto-refresh-goal]` instead of the semi-deprecated block anchor.